pages/docs/sdk/python/otel_based_sdk_v3.mdx

```diff
@@ -0,0 +1,914 @@
+async def my_async_llm_call(prompt_text):
+    # ... async LLM call ...
+    return "LLM response"
+```
+
+**Parameters:**
+
+- `name: Optional[str]`: Custom name for the created span/generation. Defaults to the function name.
+- `as_type: Optional[Literal["generation"]]`: If set to `"generation"`, a Langfuse generation object is created, suitable for LLM calls. Otherwise, a regular span is created.
+- `capture_input: bool`: Whether to capture function arguments as input. Defaults to `True`.
+- `capture_output: bool`: Whether to capture function return value as output. Defaults to `True`.
+- `transform_to_string: Optional[Callable[[Iterable], str]]`: For functions that return generators (sync or async), this callable can be provided to transform the collected chunks into a single string for the `output` field. If not provided, and all chunks are strings, they will be concatenated. Otherwise, the list of chunks is stored.
+
+**Trace Context and Special Keyword Arguments:**
+
+The `@observe` decorator automatically propagates the OTEL trace context. If a decorated function is called from within an active Langfuse span (or another OTEL span), the new observation will be nested correctly.
+
+You can also pass special keyword arguments to a decorated function to control its tracing behavior:
+
+- `langfuse_trace_id: str`: Explicitly set the trace ID for this function call.
+- `langfuse_parent_observation_id: str`: Explicitly set the parent observation ID.
+- `langfuse_public_key: str`: Use a specific Langfuse project client (useful in multi-project setups).
+
+```python
+@observe()
+def my_function(a, b):
+    return a + b
+
+# Call with a specific trace context
+my_function(1, 2, langfuse_trace_id="my_custom_trace_id_123")
+```
+
+## Integrations
+
+### OpenAI Integration
+
+Langfuse offers a drop-in replacement for the OpenAI Python SDK to automatically trace all your OpenAI API calls. Simply change your import statement:
+
+```diff
+- import openai
++ from langfuse.openai import openai
+
+# Your existing OpenAI code continues to work as is
+# For example:
+# client = openai.OpenAI()
+# completion = client.chat.completions.create(...)
+```
+
+**What's automatically captured:**
+
+- **Requests & Responses**: All prompts/completions, including support for streaming, async operations, and function/tool calls.
+- **Timings**: Latencies for API calls.
+- **Errors**: API errors are captured with their details.
+- **Model Usage**: Token counts (input, output, total).
+- **Cost**: Estimated cost in USD (based on model and token usage).
+- **Media**: Input audio and output audio from speech-to-text and text-to-speech endpoints.
+
+The integration is fully interoperable with `@observe` and manual tracing methods (`start_as_current_span`, etc.). If an OpenAI call is made within an active Langfuse span, the OpenAI generation will be correctly nested under it.
+
+**Passing Langfuse arguments to OpenAI calls:**
```

Comment:
i'd suggest to use a code example here that shows how the intgeration works within otel context